src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -330,15 +366,15 @@ private void initBindings() {
         filteredTabs.setPredicate(LibraryTab.class::isInstance);
 
         // This variable cannot be inlined, since otherwise the list created by EasyBind is being garbage collected
-        openDatabaseList = EasyBind.map(filteredTabs, tab -> ((LibraryTab) tab).getBibDatabaseContext());
+        openDatabaseList = EasyBind.map(filteredTabs, tab -> Optional.of((LibraryTab) tab).map(LibraryTab::getBibDatabaseContext).orElse(null));
         EasyBind.bindContent(stateManager.getOpenDatabases(), openDatabaseList);
 
         // the binding for stateManager.activeDatabaseProperty() is at org.jabref.gui.LibraryTab.onDatabaseLoadingSucceed
 
         // Subscribe to the search
         EasyBind.subscribe(stateManager.activeSearchQuery(SearchType.NORMAL_SEARCH), query -> {
-            if (getCurrentLibraryTab() != null) {
-                getCurrentLibraryTab().searchQueryProperty().set(query);
+            if (getCurrentLibraryTab().isPresent()) {
+                getCurrentLibraryTab().ifPresent(tab -> tab.searchQueryProperty().set(query));
```

Comment:
isPresent - if Present? This does not make any sense at all. Please do not just accept all the changes the IDE suggests.